quickwit/quickwit-ui/Makefile

```diff
@@ -4,7 +4,7 @@ build:
 	yarn build
 
 install:
-	yarn install
+	yarn install --network-timeout 1000000
```

Comment:
I had network issues during `yarn install` on the arm64 docker build. I will remove this before merging.